e2e/page-objects/addDetailPage.js

```diff
@@ -0,0 +1,102 @@
+var addDetailsCommands={
+    addComment:function(detail){
+        return this.waitForElementVisible('@chevron')
+        .click('@chevron')
+        .waitForElementVisible('@addDetailButton')
+        .click('@addDetailButton')
+        .waitForElementVisible('@addDetailDialogBox')
+        .waitForElementVisible('@addDetailArea')
+        .setValue('@addDetailArea',detail)
+        .waitForElementVisible('@saveButton')
+        .click('@saveButton')
+    },
+    assertAddDetail:function(detail){
+        return this .waitForElementVisible('@addedDetailButton')
+        .click('@addedDetailButton')
+        .waitForElementVisible('@assertDetail')
+        .assert.containsText('@assertDetail', 'Detail (1)')
+        .assert.containsText('@assertComment','Add Comment')
+    },
+    addCorrection:function(browser){
+        return this.waitForElementVisible('@chevronForCorrection')
+        .click('@chevronForCorrection')
+        .waitForElementVisible('@fileCorrection')
+        .click('@fileCorrection')
+    },
+    correctionFiling:function(browser){
+        return this
+        .assert.containsText('@correctionHeader','Correction — Annual Report (2019)')
+        .assert.containsText('@dateAssertion','Original Filing Date: 2019-03-05')
+        .assert.containsText('#correction-step-1-header','1. Detail')
+        .setValue('#detail-comment-textarea','Filed Correction')
+        .assert.containsText('#correction-step-2-header','2. Certify')
+        .setValue('#certified-by-textfield','Test')
+        .click('@certifyCheckbox')
+    },
+
+    completePyament:function(routingslip){
+        return this
+        .assert.containsText('#correction-step-3-header','3. Staff Payment')
+        .setValue('#routing-slip-number-textfield',routingslip)
+        .click('@prioritycheckbox')
+    },
+    checkFeeByIndex: function (feeName, feeValue, desiredIndex) {
+        return this
+            .assert.containsText({selector: '@feeName', index: desiredIndex}, feeName)
+            .assert.containsText({selector: '@feeValue', index: desiredIndex}, feeValue);
+    },
+    checkFeeCount: function (expectedCount) {
+        return this.expect.elements('@feeListItem').count.to.equal(expectedCount);
+    },
+    checkTotalFees: function (expectedTotalString) {
+        return this.assert.containsText('@feeTotal', expectedTotalString);
+    },
+    dashboardAssertion: function(browser){
```

Comment:
I believe you should use the existing dashboard page object for this step, since it's already present there.

Reply:
I meant the dashboardAssertion method